api/model.go

```diff
@@ -1477,6 +1477,23 @@ func toNodeResource(node *longhorn.Node, address string, apiContext *api.ApiCont
 		}
 		disks[name] = di
 	}
+	for name, status := range node.Status.DiskStatus {
```

Comment:
I know there will probably a delay of disk info between spec and status, but why we need this change in the API? any specific purpose? I feel this will cause some confusion.

Reply:
Yeah, this is a change for the regression test.
For example, after doing `node.diskUpdate`, get the node and check the disk spec and status and see if the update is successful. If not, retry `node.diskUpdate`.
I think this change is confusing as well, so I'll drop this commit and see if there is better way on the regression test side.